### PR TITLE
Added the build matrix to the build started comment

### PR DESCRIPTION
**Pull request by oleorhagen**
This adds a simple markdown table with the build variables used for this
particular build.

Handy to make sure that the build is from a cross-repo build or not.

Changelog: None

Signed-off-by: Ole Petter <ole.orhagen@northern.tech>

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -363,12 +363,43 @@ func triggerBuild(conf *config, build *buildOptions, pr *github.PullRequestEvent
...
 	}
 	log.Infof("Created pipeline: %s", pipeline.WebURL)
 
+	// Add the build variable matrix to the pipeline comment under a
+	// drop-down tab
+	tmplString := `
+Hello :smile_cat: I created a pipeline for you here: [Pipeline-.Pipeline.ID](.Pipeline.WebURL)
+
+<details>
+    <summary>Build Configuration Matrix</summary><p>
```
Comment (lluiscampos) [inline main.go:376]: Shouldn't this be markdown?

**Comment by lluiscampos** [comment]
It looks good at first glance. Do you have any PR with this type of comment?

**Comment by oleorhagen** [comment]
> It looks good at first glance. Do you have any PR with this type of comment?

Yes, it is the same thing done in: 
https://github.com/mendersoftware/deployments-enterprise/pull/106#issuecomment-613999208

And here the template is typed out:

---------
Hello :smile_cat: I created a pipeline for you here: [Pipeline-.Pipeline.ID](.Pipeline.WebURL)

<details>
    <summary>Build Configuration Matrix</summary><p>

| Key   | Value |
| ----- | ----- |
{{range $i, $var := range .BuildVars.Variables}}
| {{var.Key}} | {{var.Value}} |
{{end}}

 </p></details>

File: main.go (hunk 1 above)
Comment (oleorhagen) [reply main.go:376]: Nope, I need html for this

Comment (lluiscampos) [reply main.go:376]: k

**Comment by oleorhagen** [comment]
Example test-run:
https://github.com/mendersoftware/meta-mender/pull/973#issuecomment-616702540

**Comment by kacf** [comment]
Nice! I didn't know about the `<details>` tag.

**Pull request merged**